Lookuptable/settings-sample.cfg

```diff
@@ -0,0 +1,12 @@
+[default]
+# This is the environment url of commcare
+url = https://www.commcarehq.org/
+# Login username of the webuser
+login_username =
+# Login password of the webuser
+login_password =
+# This is a preconfigured browsertack username for mobile tests
+staging_auth_key =
+# This is a preconfigured authentication key used for 2FA tests on prod
+prod_auth_key =
```

Comment:
Are we making use of these?
Also, the description says `# This is a preconfigured BrowserStack username for mobile tests`. I don't think we're doing any mobile-related tests here.

Reply:
Changes done. Commit id: 0d84b418e608b3130b5c249c9d1f9a93b7b7c14f